### PR TITLE
Issue#733 fixed - Added QR Generator Project. 

### PR DESCRIPTION
**Pull request by ParnaRoyChowdhury777**
## Related Issue

#733 

## Description

Issue#733 fixed - I added QR Generator Project under Showcase section. 

## Type of PR

- [x] Feature enhancement

## Screenshots / Videos (if applicable)

Screenshot 2024-06-08 013114

## Checklist
- [x] I have performed a self-review of my code.
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.

## Resources for Guidance

Here are some resources that may be helpful as you contribute to CodeHarborHub:
- [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [Markdown Guide](https://www.markdownguide.org/)
- [MDX Documentation](https://mdxjs.com/docs/)
- [Mermaid Documentation](https://mermaid.js.org/)

**Resources for DSA Features on Our Site:**

- Tabs: [Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex)

### CODE REVIEW

File: src/data/users.tsx [hunk 1]
```diff
@@ -292,6 +292,14 @@ const Users: User[] = [
     source: "https://github.com/ParnaRoyChowdhury777/Guess_The_Number_Game",
     tags: ["opensource", "product", "html", "css", "javascript", "vanilla"],
   },
+  {
+    title: "QR Generator",
+    description: "It generates a QR code based on a typed text or URL.",
+    preview: require("./showcase/qr-generator.png"),
```
Comment (Ajay-Dhangar) [inline src/data/users.tsx:298]: replace `qr-generator.png` to `qr_generator.png`

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
update your code based on suggestions

**Comment by ParnaRoyChowdhury777** [comment]
@Ajay-Dhangar Changes made

**Review by Ajay-Dhangar — APPROVED** [review]
done

**Pull request merged**